redash/models/users.py

```diff
@@ -334,31 +358,34 @@ def grant(cls, obj, access_type, grantee, grantor):
 
     @classmethod
     def revoke(cls, obj, grantee, access_type=None):
-        permissions = cls._query(obj, access_type, grantee)
-        return permissions.delete()
+        conditions = AccessPermission._query_condition(obj, access_type, grantee, None)
+        q = delete(cls).where(*conditions)
+        return db.session.execute(q).rowcount
 
     @classmethod
     def find(cls, obj, access_type=None, grantee=None, grantor=None):
         return cls._query(obj, access_type, grantee, grantor)
 
     @classmethod
     def exists(cls, obj, access_type, grantee):
-        return cls.find(obj, access_type, grantee).count() > 0
+        return len(cls.find(obj, access_type, grantee)) > 0
```

Comment:
It has been changed from `count()` to `len()`

Reply:
updated this as well

Reply:
Thanks. Confirmed.